package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "@mastodon/mastodon",
   "license": "AGPL-3.0-or-later",
+  "packageManager": "yarn@4.0.1",
```

Comment:
I've just moved this up, such that it's near the `engines` field and not forgotten about below the dependencies